### DISCUSSION TITLE
Pot | Anki

### DISCUSSION THREAD

**Discussion by giscus** [pot-app/pot-desktop#823] [General]
# Pot | Anki

跨平台划词翻译和OCR软件

https://pot-app.com/docs/api/collection/anki.html

**Comment by 306686767** [comment]
添加生词时,提示添加成功,但ANKI中没有,不知加到哪个牌组里了

**Reply by 1613358894** [reply]
一样的问题，显示成功，但实际上anki中没有。

**Comment by oaoming** [comment]
能说下怎么配置POT和anki的连接么？
添加后添加道anki哪里的牌组里了？
用的什么模板，
注意事项是啥，上面都没写全

**Comment by zhi-11** [comment]
添加到anki后，单词及句子的发音没有一并添加
